examples/network/git2.c

```diff
@@ -57,16 +49,42 @@ int main(int argc, char **argv)
 
 	git_libgit2_init();
 
+	struct args_info args = ARGS_INFO_INIT;
+	const char *git_dir = NULL;
+
+	for (args.pos = 1; args.pos < args.argc; ++args.pos) {
+		char *a = args.argv[args.pos];
+
+		if (a[0] != '-') {
+			/* non-arg */
+			break;
+		} else if (optional_str_arg(&git_dir, &args, "--git-dir", ".git")) {
```

Comment:
Is the empty body for this if statement also left intentionally empty?

Reply:
Yeah, a `continue` would be nice to have here